### ISSUE TITLE
Mysqli connection can be done on the wrong return?

### ISSUE THREAD

**Issue by ejcms** [joshcam/PHP-MySQLi-Database-Class#689]
When mysqli given the connection information is wrong, return mysqli connection error information.
![default](https://user-images.githubusercontent.com/31702375/30744637-906a880a-9fd5-11e7-8518-4f35edd02b13.jpg)

**Comment by avbdr** [comment]
can you rephrase your question?

**Comment by ejcms** [comment]
If the input information is wrong when connect to mysqli, is it OK to add a return error message?

**Comment by avbdr** [comment]
there is no if check, but there is an excetion you can catch
```php
<?php
require_once ("../MysqliDb.php");

$db = new Mysqlidb('localhost', 'i1root', '', 'testdb');
try {
    $db->ping();
} catch (Exception $e) {
    echo $e->message;
}
?>
```

**Comment by avbdr** [comment]
as well note that contructor wont try to establish any connection. You will need use ping() or any get() operation to establish a connection.

**Comment by ejcms** [comment]
alright, thank you very much!
There is aa query, the query field to give the value of 0, the query will not show the field.
Such as: $db->where("testii", 0)->get("table");
Print: SELECT  * FROM table
Can this query: SELECT  * FROM table WHERE testii = '0'
